thoth/storages/graph/postgres.py

```diff
@@ -3953,7 +3953,7 @@ def sync_analysis_result(self, document: dict) -> None:
         environment_type = environment_type.upper()
         origin = document["metadata"]["arguments"]["thoth-package-extract"]["metadata"].get("origin")
         environment_name = document["metadata"]["arguments"]["extract-image"]["image"]
-        os_name = document["result"]["operating-system"]["name"]
```

Comment:
Didn't we wanted to have the same structure for all components?, like adviser, inspection, etc which uses `operating_system` and `name`. the `id` is given from `os_release`

Reply:
`os_release` comes from generic analyzer metadata - package-extract has "os_info" handler for this

https://github.com/thoth-station/package-extract/blob/93b2800066114750700e4dcd9c5f167b246c6064/thoth/package_extract/image.py#L606-L607

Under the hood, they use similar code now. I wanted to sync this from "result", not from package-extract "metadata".

Reply:
Thanks for the clarification, then LGTM!! :100: